example/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 include(CheckCXXCompilerFlag)
 
 if((CMAKE_CXX_COMPILER_ID MATCHES "GNU") OR (CMAKE_CXX_COMPILER_ID MATCHES "Clang"))
-    set(OPTIONS -Wall -Wextra -pedantic-errors -Werror)
+    set(OPTIONS -Wall -Wextra -pedantic-errors -Werror -std=c++17)
```

Comment:
Perhaps a better approach would be to use `set(CMAKE_CXX_STANDARD 17)`.

Reply:
Probably, yeah. I cannot test it thought due to #244

Reply:
@embeddedpenguin  https://github.com/Neargye/magic_enum/issues/244 fixed in master

Reply:
> Perhaps a better approach would be to use `set(CMAKE_CXX_STANDARD 17)`.

Updated to use `set(CMAKE_CXX_STANDARD 17)`.